Python/other/Possible_Code_strings.py

```diff
@@ -0,0 +1,38 @@
+# Given a string s, and assuming that a=1, b=2, c=3, d=4, .... z=26
+# Python recursive function to print all the possible codes for the strings.
+
+
+def output(original, previous):
+
+    if len(original) == 0:  # if length is zero print 0th value i.e., '`'
+        print(previous)
+        return
+    if len(original) == 1:
+        print (previous + chr(96 + int(original)))  # checking range
+        return
+
+    if int(original[0:2]) < 27:
+        new_character = chr(96 + int(original[0:2]))  # getting  new character
+        output(original[2:], previous + new_character)
+    new_character = chr(96 + int(original[0:1]))
+    output(original[1:], previous + new_character)
+
+# driver code
+if __name__ == '__main__':
```

Comment:
try to set a fixed way/template to write your program

Your code seems to be copied from some coding platforms.

Reply:
repository don't exist from where this PR created, @plazzy99 sir, you closed my new PR.

Reply:
@plazzy99 sir, kindly close this issue, because from which branch it is created, it don't exist, and neither its repo. I'm creating new PR. Kindly look at that PR. Thank you!

Reply:
> try to set a fixed way/template to write your program
> 
> Your code seems to be copied from some coding platforms.

i formatted the code according to pep-8 standard